src/raft/raft.rs

```diff
@@ -1042,6 +1056,26 @@ impl<T: Storage> Raft<T> {
                 self.send(to_sent);
                 return;
             }
+            MessageType::MsgReadIndex => {
+                let mut read_index = 0;
+                if self.check_quorum {
+                    read_index = self.raft_log.committed
+                }
+                if m.get_from() == 0 || m.get_from() == self.id {
```

Comment:
@siddontang 
Is there any constant var for something like `INVALID_RAFT_LOG_INDEX` already?